assets/app/view/user.rb

```diff
@@ -2,18 +2,22 @@
 
 require 'game_manager'
 require 'user_manager'
+require 'lib/settings'
 require 'view/game_row'
 require 'view/logo'
 require 'view/form'
 
 module View
   class User < Form
-    include Lib::Color
+    include Lib::Settings
     include GameManager
     include UserManager
 
     needs :type
 
+    DARK = `window.matchMedia('(prefers-color-scheme: dark)').matches`.freeze
```

Comment:
why is this copied from settings?

Reply:
It’s the other way round, but obsolete anyway – just like a bit more.